packages/design-tokens/tokens/spacing.yml

```diff
@@ -5,15 +5,24 @@ props:
   - name: spacing-800
     value: 4rem
     comment: 64px
+  - name: spacing-750
+    value: 3.5rem
+    comment: 56px
   - name: spacing-700
     value: 3rem
     comment: 48px
+  - name: spacing-650
+    value: 2.5rem
+    comment: 40px
   - name: spacing-600
     value: 2rem
     comment: 32px
   - name: spacing-500
```

Comment:
Was thinking about adding a `550` here just to be consistent

Reply:
👍 I think that would be a good idea - might be confusing otherwise.